Fix:
conftest.c:105:21: error: implicit declaration of function 'sendfile' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2021-12-04 08:40:40.000000000 -0600
+++ configure	2021-12-05 07:34:33.000000000 -0600
@@ -20446,6 +20446,9 @@
         #ifdef HAVE_SYS_SENDFILE_H
         #include <sys/sendfile.h>
         #endif /* HAVE_SYS_SENDFILE_H */
+        #include <sys/types.h>
+        #include <sys/socket.h>
+        #include <sys/uio.h>
         #include <errno.h>
         int main() {
           int o = 0;
